src/engraving/dom/score.cpp

```diff
@@ -466,8 +466,10 @@ void Score::setUpTempoMap()
             BeatsPerSecond currentBps = tempomap()->tempo(tickPositionFrom);
             BeatsPerSecond newBps = currentBps * tempoChange->tempoChangeFactor();
 
-            std::map<int, double> tempoCurve = TConv::easingValueCurve(tempoChange->ticks().ticks(),
-                                                                       4 /*stepsCount*/,
+            int totalTicks = tempoChange->ticks().ticks();
+            int stepsCount = std::max(8, totalTicks / 480);
```

Comment:
8 seems a sensible minimum for tempo changes over short periods (up to 2 4/4 measures).  For longer tempo changes, ensure there's an adjustment at least every 1/4 note beat - not theoretically ideal for compound meters but it sounds fine in practice.